modules/about-must-gather.adoc

```diff
@@ -13,11 +13,12 @@
 The `oc adm must-gather` CLI command collects the information from your cluster that is most likely needed for debugging issues, such as:
 
 * Resource definitions
-* Audit logs
 * Service logs
 
 You can specify one or more images when you run the command by including the `--image` argument. When you specify an image, the tool collects data related to that feature or product.
 
+You can alternatively collect audit logs using the `-- /usr/bin/gather_audit_logs` flag. Audit logs are not collected as part of the default set of information to reduce the size of the files. 
+
```

Comment:
Hi @mburke5678 , I think it would be better to move this paragraph to a bottom NOTE section instead, how about something like this?
~~~
[NOTE]
====
Audit logs are no longer gathered by default to reduce the folder size, for more info please refer to section xref:../support/gathering-cluster-data.html#gathering-data-audit-logs_gathering-cluster-data[Gathering audit logs].
====
~~~
NOTE: please double check the xref path, not sure if I have used it correctly.

Reply:
@pamoedom Good idea to use a link. However, we cannot add links to modules (for reasons that are not clear to me). I added a _as described in the following sections_. And reorganized a lit to tighten up the sentences, particularly the sentence on the --image argument.